ported/omb/rdm_latency_threaded

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
That's fine. I'm providing these PRs in part to demonstrate the changes I
would do for 'thread_completion' measurements.

-- Jim




On 4/14/16, 11:38 AM, "Sung-Eun Choi" <notifications@github.com> wrote:

>In 
>ported/omb/rdm_latency_threaded
><https://github.com/ofi-cray/fabtests-cray/pull/74#discussion_r59749718>:
>> @@ -0,0 +1,13 @@
>> +#!/bin/bash
>Once things have settled down, we can change the test scripts.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/fabtests-cray/pull/74/files/8f0a955ac1789fd1b
>cd62ae28717540167964d72#r59749718>
>
>